labgrid/driver/mqtt.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3
+
+import queue
+import logging
+import threading
+import time
+
+import attr
+import paho.mqtt.publish as publish
+import paho.mqtt.client as mqtt
+
+from .common import Driver
+from ..factory import target_factory
+from ..protocol import PowerProtocol
+from ..step import step
+from ..util import Timeout
+
+
+@target_factory.reg_driver
+@attr.s(eq=False)
+class TasmotaPowerDriver(Driver, PowerProtocol):
+    bindings = {
+            "power": {"TasmotaPowerPort", "NetworkTasmotaPowerPort"}
```

Comment:
We talked about this in a short call and agreed that the `NetworkTasmotaPortPort` does not make much sense. The exporter should export a plan `TasmotaPowerPort` instead, which gets instanciated again on the client side.